tasks/less.js

```diff
@@ -169,10 +169,14 @@ module.exports = function(grunt) {
 
   var parseVariableOptions = function(options) {
     var pairs = _.toPairs(options);
-    var output = '';
-    pairs.forEach(function(pair) {
-      output += '@' + pair[0] + ':' + pair[1] + ';';
-    });
+    var output = _.reduce(pairs, function(accumulator, pair) {
+      var variableName = pair[0];
+      var variableValue = pair[1];
+      if (variableValue !== undefined && variableValue !== null) {
```

Comment:
Can tyou change this to `typeof !== 'undefined'`? Also, is the check for `null` needed here?